package_index.go

```diff
@@ -60,15 +60,15 @@ func packageIndexHandler(packagesBasePaths []string, cacheTime time.Duration) fu
 		w.Header().Set("Content-Type", "application/json")
 		cacheHeaders(w, cacheTime)
 
-		aPackage, err := util.NewPackageWithResources(packagePath)
+		p, err := util.NewPackageWithResources(packagePath)
```

Comment:
I'd rather stick to this explanation: https://www.codereadability.com/i-n-k-single-letter-variable-names/

Reply:
I rather stick to consistency across the registry code for now. 

I would add to this list when using inside a method to reference the type like `func (p Package) ...`. But yes, personal preference.